internal/kibana/policies.go

```diff
@@ -52,7 +52,7 @@ func (c *Client) CreatePolicy(p Policy) (*Policy, error) {
 func (c *Client) DeletePolicy(p Policy) error {
 	reqBody := `{ "agentPolicyId": "` + p.ID + `" }`
 
-	statusCode, respBody, err := c.post("agent_policies/delete", []byte(reqBody))
+	statusCode, respBody, err := c.post("/api/fleet/agent_policies/delete", []byte(reqBody))
```

Comment:
What do you think about extracting all URL prefixes to the common place? I know it's an improvement comparing to what we have now.

Reply:
Sure, but just want to clarify what you mean by URL prefixes: just the `/api/fleet` part (and similar prefixes from other API URLs) or something more?

Reply:
Yes, I meant exactly that: '/api/fleet' or '/api/saved_objects' etc. Just to prevent missing mappings in the future (because they were hidden in multiple places in the source).

Reply:
Done in 17ab10c70570def207fd49eb186875a0d941b258.